src/test/ui/lint/issue-71290-unused-paren-binop.rs

```diff
@@ -0,0 +1,17 @@
+// check-pass
+// Make sure unused parens lint doesn't emit a false positive.
+// See https://github.com/rust-lang/rust/issues/71290 for details.
```

Comment:
```suggestion
// See https://github.com/rust-lang/rust/issues/71290 for details.
#![deny(unused_parens)]
```
`ui/lint` tests set all lints to `allow` afaik, so this test already passes on the current beta as is.

Reply:
Thanks :)